scripts/extract_location.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
```

Comment:
You should be able to generate location insights using:
`python3 -m robotoff ocr.jsonl.gz -t location -o location.jsonl`

Reply:
So I don't think we need this script